### PR TITLE
klog messages

### LINKED ISSUE #64: use klog instead of log
/kind feature

**Describe the solution you'd like**
cluster-api uses `klog`. So we should also use it to avoid mixed log functions

### PR DESCRIPTION
**Pull request by xunpan**
Fixes #64

### CODE REVIEW

File: pkg/cloud/ibmcloud/actuators/cluster/actuator.go [hunk 1]
```diff
@@ -43,13 +41,17 @@ func NewActuator(params ActuatorParams) (*Actuator, error) {
 
 // Reconcile reconciles a cluster and is invoked by the Cluster Controller
 func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
-	log.Printf("Reconciling cluster %v.", cluster.Name)
-	// "TODO: Not yet implemented"
+	klog.Infof("Reconciling cluster %v.", cluster.Name)
+
+	klog.Infof("Reconciling cluster %v: Not implemented yet.", cluster.Name)
```
Comment (jichenjc) [inline pkg/cloud/ibmcloud/actuators/cluster/actuator.go]: I assume you have those 2 on purpose ,right?

File: pkg/cloud/ibmcloud/actuators/machine/actuator.go [hunk 2]
```diff
@@ -160,31 +159,32 @@ func (ic *IbmCloudClient) Delete(ctx context.Context, cluster *clusterv1.Cluster
...
+	klog.Infof("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
+
 	klog.Infof("TODO: Not yet implemented")
 	return nil
 }
 
 // Exists test for the existance of a machine and is invoked by the Machine Controller
 func (ic *IbmCloudClient) Exists(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) (bool, error) {
-	guest, err := ic.guestExists(machine)
+	_, err := ic.guestExists(machine)
```
Comment (jichenjc) [inline pkg/cloud/ibmcloud/actuators/machine/actuator.go]: I assume the function will return NOTFOUND error in err, right?

File: pkg/cloud/ibmcloud/clients/machines.go [hunk 3]
```diff
@@ -73,28 +76,34 @@ func NewInstanceServiceFromMachine(kubeClient kubernetes.Interface, machine *clu
...
-		log.Print(".")
 		// TODO(gyliu513) make it configurable or use the notification mechanism to optimize
 		// the process instead of hardcoded waiting.
 		time.Sleep(5 * time.Second)
 		transactions, _ = s.GetActiveTransactions()
 	}
-	log.Println("wait done")
+	s.Session.Debug = false
+
+	klog.Info("Waiting for trasactions <%D> done.")
```
Comment (jichenjc) [inline pkg/cloud/ibmcloud/clients/machines.go]: where's %D?

Comment (xunpan) [reply pkg/cloud/ibmcloud/clients/machines.go]: Type. I fixed it

File: pkg/cloud/ibmcloud/actuators/machine/actuator.go (hunk 2 above)
Comment (xunpan) [reply pkg/cloud/ibmcloud/actuators/machine/actuator.go]: This function only returns bool. It does not need `guest`. And `guestExists` always return `error` if there is no host found

File: pkg/cloud/ibmcloud/actuators/cluster/actuator.go (hunk 1 above)
Comment (xunpan) [reply pkg/cloud/ibmcloud/actuators/cluster/actuator.go]: I will change it to `TODO` for second message

**Pull request merged**